.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Test deploy
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  build:
+    if: ${{ github.event.label.name == 'deploy' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Deploy
+        shell: bash
+        run: npx vercel --force --token "$VERCEL_TOKEN"
```

Comment:
maybe we want to take the output from this and add it as a comment? not a big deal though as we can always click through to get the url